config/rewrites.d/developer.rackspace.com.json

```diff
@@ -33,6 +33,13 @@
             "status": 301
         },
         {
+            "from": "^\\/signup",
```

Comment:
@smashwilson just the one path: https://github.com/rackerlabs/docs-developer-blog/tree/master/signup